custom/icds_reports/templates/icds_reports/icds_app/download.directive.html

```diff
@@ -56,6 +56,21 @@
                 </div>
             </div>
             <div ng-if="!$ctrl.isISSNIPMonthlyRegisterSelected()">
+
+                <div class="form-group" ng-if="($ctrl.isSDRSelected())">
+                    <label class="control-label col-sm-2" for="format">Beneficiary: <i class="fa fa-info-circle" uib-popover="Select the category of beneficiaries"
```

Comment:
nit: should this be "select the category of *beneficiary*"?

Reply:
Yeah right